src/alpine/3.13/amd64/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM alpine:3.13
+
+RUN apk update
+
+RUN apk add --no-cache \
+        autoconf \
+        automake \
+        bash \
+        build-base \
+        clang \
+        clang-dev \
+        coreutils \
+        curl \
+        curl-dev \
+        gcc \
+        gettext-dev \
+        git \
+        icu-dev \
+        krb5-dev \
+        libtool \
+        libunwind-dev \
+        linux-headers \
+        llvm \
+        make \
+        openssl \
+        openssl-dev \
+        paxctl \
+        python \
```

Comment:
`python` doesn't exist in Alpine 3.13.  It actually didn't exist in Alpine 3.9 either but was configured to be replaced by `python2` by default.  That replacement configuration doesn't exist in Alpine 3.13, so this needs to explicitly reference `python2` instead.